.github/workflows/mariadb.yml

```
@@ -0,0 +1,78 @@
+name: Build Mroonga with MariaDB main
```

MroongaのリポジトリーでMroongaをビルドするのは普通なのでもっとシンプルにできませんか？

```suggestion
name: MariaDB
```